polynote-frontend/polynote/ui.js

```diff
@@ -1471,7 +1500,7 @@ export class MainUI extends EventTarget {
         if (!this.welcomeUI) {
             this.welcomeUI = new WelcomeUI().setEventParent(this);
         }
-        this.tabUI.addTab('home', 'Home', { notebook: this.welcomeUI.el, path: '/' }, 'home');
+        this.tabUI.addTab('home', span([], 'Home'), { notebook: this.welcomeUI.el, path: '/' }, 'home');
```

Comment:
Fixes a small bug in the Home screen - the tab looks funky because it's missing this inner `span`